src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -1503,35 +1481,23 @@ private bool CanExecuteActiveCode()
             return _evaluator.CanExecuteCode(input);
         }
 
-        #endregion
+#endregion
 
-        #region Buffers, Spans and Prompts
-        private ITrackingSpan CreateStandardInputPrompt()
+#region Buffers, Spans and Prompts
+        private object CreateStandardInputPrompt()
         {
-            return CreateTrackingSpan(_standardInputPromptBuffer, string.Empty);
+            return string.Empty;
         }
 
-        private ITrackingSpan CreatePrimaryPrompt()
+        private object CreatePrimaryPrompt()
```

Comment:
It seems strange that ```CreatePrimaryPrompt``` and ```CreateSecondaryPrompt``` have the same implementation.  Maybe they should be merged into ```CreatePrompt```?